tools/workbook-generator/.gitignore

```
@@ -0,0 +1,5 @@
+__pycache__/**
+data/ay22/*.sqlite3
+output/**
+templates/*.xlsx
+.~lock*
```

```suggestion
.~lock*

```
Same thing here, new line at eof to remove the `-`